components/psst/BUILD.gn

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+
+static_library("psst") {
```

Comment:
Using `static_library` is just a workaround for existing components. New components should use `component` type with appropriate exports for `is_component = true`

Reply:
>with appropriate exports for is_component = true

WDYM?